odoo/tools/config.py

```diff
@@ -369,7 +369,7 @@ def __init__(self, fname=None):
         # generate default config
         self._parse_config()
 
-    def parse_config(self, args=None):
+    def parse_config(self, args=None, *, setup_logging=False):
```

Comment:
Most of the time, we want to have logging set up according to the configuration. You could opt out if necessary.
```suggestion
    def parse_config(self, args=None, *, setup_logging=True):
```